package.json

```diff
@@ -69,7 +69,7 @@
     "prettier": "prettier --write 'packages/**/*.ts'",
     "lint": "eslint . --ext .ts",
     "release": "yarn && lerna publish --no-private --force-publish && yarn retype:updateversion",
-    "release:ci": "yarn && lerna publish --no-private --force-publish --yes --no-verify-access && yarn retype:updateversion",
```

Comment:
- don't need to re-run `yarn` in ci
- `--no-verify-access` is now the default
- don't need to update retype docs version (because we don't publish docs in this ci step)